utils/genesis/src/lib.rs

```diff
@@ -143,7 +144,12 @@ where
         info!("Reading file...");
         let file = File::open(&path).await?;
         let reader = FetchProgress::fetch_from_file(file).await?;
-        load_and_retrieve_header(sm.blockstore(), reader, skip_load).await?
+        {
+            let stopwatch = time::Instant::now();
+            let cids = load_and_retrieve_header(sm.blockstore(), reader, skip_load).await?;
+            info!("Loaded .car file in {}s", stopwatch.elapsed().as_secs());
+            cids
+        }
```

Comment:
Not a huge deal, but you are measuring the time only for the local file branch. For consistency, it should also be measured for the remote file branch above (L138-142). It should be completely fine to include parsing `Url` and opening files to simplify logic, as those operations' overhead is negligible compared to loading the snapshot.

Reply:
Good point, will fix this.